<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAddEdit</class>
 <widget class="QDialog" name="DialogAddEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add anime</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="HLay_0">
     <item>
      <layout class="QVBoxLayout" name="VLay_0">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <layout class="QHBoxLayout" name="HLay_1">
         <item>
          <spacer name="HSpacer_0">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="CheckBox_Editing">
           <property name="text">
            <string>Editing</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="LineEdit_Title">
         <property name="maxLength">
          <number>128</number>
         </property>
         <property name="placeholderText">
          <string>Title</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="VLay_OrigTitle">
         <property name="topMargin">
          <number>0</number>
         </property>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="HLay_DirectorAndSound">
         <property name="topMargin">
          <number>0</number>
         </property>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="Lbl_Year">
           <property name="text">
            <string>Year:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="SpinBox_Year">
           <property name="minimum">
            <number>1900</number>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
           <property name="value">
            <number>2000</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="HSpacer_1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTabWidget" name="TabWidget_Info">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>Containing music</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLineEdit" name="LineEdit_Tags_2">
               <property name="toolTip">
                <string>Separator: comma</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="maxLength">
                <number>256</number>
               </property>
               <property name="placeholderText">
                <string>Enter music title</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_4">
               <property name="toolTip">
                <string>Add in the list</string>
               </property>
               <property name="text">
                <string>+</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_5">
               <property name="toolTip">
                <string>Delete selected</string>
               </property>
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QListView" name="listView"/>
           </item>
          </layout>
          <zorder>LineEdit_Tags_2</zorder>
          <zorder>toolButton_4</zorder>
          <zorder>toolButton_5</zorder>
          <zorder></zorder>
          <zorder>listView</zorder>
         </widget>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Containing anime</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPlainTextEdit" name="plainTextEdit">
             <property name="placeholderText">
              <string>Containing anime</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Author comment</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPlainTextEdit" name="PlainTextEdit_Description">
             <property name="placeholderText">
              <string>Author comment</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>Media</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="HLayout_Dir">
             <item>
              <widget class="QLineEdit" name="LineEdit_Dir">
               <property name="maxLength">
                <number>256</number>
               </property>
               <property name="placeholderText">
                <string>Directory</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton">
               <property name="toolTip">
                <string/>
               </property>
               <property name="text">
                <string>*</string>
               </property>
               <property name="icon">
                <iconset resource="resource.qrc">
                 <normaloff>:/images/folder.png</normaloff>:/images/folder.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLineEdit" name="LineEdit_URL">
             <property name="maxLength">
              <number>256</number>
             </property>
             <property name="placeholderText">
              <string>URL</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="VSpacer_0">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>138</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="ImageCover" name="Lbl_ImageCover">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>3</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>540</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>275</width>
         <height>825</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Drop image here, or a double click for a file choice</string>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resource.qrc">:/images/DropHere.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::NoTextInteraction</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="BtnBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Reset|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageCover</class>
   <extends>QLabel</extends>
   <header location="global">imagecover.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
